modules/sdk/gradle-plugins-defaults/CHANGELOG.markdown

```diff
@@ -380,12 +380,26 @@ following system properties to `true`:
 ### Changed
 - [LPS-68334]: Update the [Liferay Gradle Plugins] dependency to version 2.0.50.
 
+## 1.2.44 - 2016-10-06
+
+### Changed
+- [LPS-66396]: Update the [Liferay Gradle Plugins] dependency to version 3.0.0.
+
+### Fixed
+- [LPS-66396]: Remove dependency from Java 8 class.
```

Comment:
@Ithildir do you think we could replace that with the following:

> [LPS-66396]: Update import in several classes from
`java.io.UncheckedIOException` to `org.gradle.api.UncheckedIOException`.

Or is there another choice of words that would be better?